js/load-scene.js

```diff
@@ -61,23 +61,56 @@ AFRAME.registerComponent('loadscene', {
       lightPop.setAttribute('position', { x: -3.026, y: 4.122, z: -0.366 });
     }
 
+    var containerEl = document.getElementById('sts-live');
+    var sceneEl = document.querySelector('a-scene');
+    var introAudioEl = document.getElementById('intro-audio');
+    var ambienceAudio = document.getElementById('ambience');
+    var getReady = document.getElementById('get-ready');
+    var getStartedButton = document.getElementById('get-started');
 
-    document.querySelector('a-scene').addEventListener('loaded', function () {
-      document.getElementById('get-ready').style.display = 'none';
-      document.getElementById('intro-audio').play();
-      document.getElementById('ambience').play();
-      document.getElementById('camera-audio').addEventListener('loaded', function() { console.log('ok') })
+    var enterVREl;
+    var audioStarted = false;
+    var userPressEvent = 'ontouchstart' in window ? 'touchend' : 'mousedown';
+    var userPressDown = onUserPressDown.bind(this);
+    var sceneLoaded = onSceneLoaded.bind(this);
+
+    function onUserPressDown() {
+      if (audioStarted) {
+        return;
+      }
+      introAudioEl.play();
+      ambienceAudio.play();
+      audioStarted = true;
+
+      if (enterVREl) {
+        enterVREl.removeEventListener(userPressEvent, userPressDown);
+      }
+      sceneEl.removeEventListener(userPressEvent, userPressDown);
+    }
+
+    function onSceneLoaded() {
+      enterVREl = sceneEl.components['vr-mode-ui'].enterVREl;
+      getReady.style.display = 'none';
+      introAudioEl.load();
+      ambienceAudio.load();
+      // start audio after geseture on enter VR button.
+      enterVREl.addEventListener(userPressEvent, userPressDown);
+      sceneEl.removeEventListener('loaded', onSceneLoaded);
+      browserReposition();
+    }
+
+    // start audio after user gesture on scene.
+    sceneEl.addEventListener(userPressEvent, userPressDown);
+    sceneEl.addEventListener('loaded', sceneLoaded);
```

Comment:
with this change, even after clicking the Enter VR button, once I am in VR mode, I have to click again for the scene graphics to show up. (though the **Get Started** and score counter still show up against a black background.)

this confused me, and I reloaded and retried refreshing the page multiple times, thinking it was broken. can we listen on the first `click` event on the **Enter VR** button if we have `navigator.getVRDisplays`, and then the first `touchstart`/`click` otherwise?

Reply:
I'm unsure what you mean by "scene graphics to show up"?  It seems to work for me.